pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -29,6 +30,13 @@ in stdenv.mkDerivation rec {
     sha256 = "sha256-Of6mS1kMlJKYSgwnqJ/CA+HNx0hmCG77j0aYZ3qytXQ=";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Should probably have a comment that it's required for the musl build to succeed, so it's clear when it can be removed.

Reply:
Done